pyanaconda/ui/lib/help.py

```diff
@@ -237,20 +237,20 @@ def _find_best_help_file(current_locale, available_files):
 
 
 def _get_help_user():
-    """Get the uid to run help viewer under.
+    """Get the user name and uid to run help viewer under.
 
     Currently, this means: Let's see if we can run yelp under liveuser on live.
     For details, see: https://bugzilla.redhat.com/show_bug.cgi?id=2118832
 
-    :return: user
-    :rtype: int | None
+    :return: user name and uid
+    :rtype: (str, int) | None
```

Comment:
I would never recommend mixing types like this, unless there is a very good reason for it. Let's raise an exception instead of returning `None`. It will a more Pythonic solution.

Reply:
Actually, let's use some structured type to handle the data, so that it can be None as usual.